cpp/src/IceUtil/UtilException.cpp

```diff
@@ -54,15 +54,11 @@
 //
 // The Slice compilers don't retrieve the exception stack traces so we don't need the DbgHelp calls.
 //
-#if defined(_WIN32) && !defined(__MINGW32__) && !defined(ICE_BUILDING_SLICE_COMPILERS)
+#if defined(_WIN32) && !defined(ICE_BUILDING_SLICE_COMPILERS)
 #   define ICE_DBGHELP
-#   if defined(_MSC_VER) && (_MSC_VER >= 1700)
-#       define DBGHELP_TRANSLATE_TCHAR
-#       include <IceUtil/StringConverter.h>
-#       if _MSC_VER >= 1900
-#           pragma warning(disable:4091) // VS 2015 RC issues this warning for code in DbgHelp.h
-#       endif
-#   endif
+#   define DBGHELP_TRANSLATE_TCHAR
+#   include <IceUtil/StringConverter.h>
+#   pragma warning(disable:4091) // VS 2015 RC issues this warning for code in DbgHelp.h
```

Comment:
Can you add a TODO for this one, not clear it is required with Visual Studio 2022